### PR TITLE
New PR for Read receipt changes

### PR DESCRIPTION
**Pull request by Jegadeesh-MSFT**

### CODE REVIEW

**Comment by Jegadeesh-MSFT** [comment]
**Executed Build Pipeline**

https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/6295051288

image

**Comment by ChetanSharma-msft** [comment]
image

File: samples/bot-conversation/csharp/AppManifest/manifest.json [hunk 1]
```diff
@@ -68,5 +68,19 @@
     "identity",
     "messageTeamMembers"
   ],
+  "webApplicationInfo": {
+    "id": "<<YOUR-MICROSOFT-APP-ID>>",
+    "resource": ""
```
Comment (Prithvi-MSFT) [inline samples/bot-conversation/csharp/AppManifest/manifest.json:73]: Can we add demo string here. Plese refre graph rsc sample

File: samples/bot-conversation/csharp/Bots/TeamsConversationBot.cs [hunk 2]
```diff
@@ -175,6 +249,7 @@ private async Task DeleteCardActivityAsync(ITurnContext<IMessageActivity> turnCo
```
Comment (Prithvi-MSFT) [inline samples/bot-conversation/csharp/Bots/TeamsConversationBot.cs:249]: Add method summary

File: samples/bot-conversation/csharp/AppManifest/manifest.json [hunk 3]
```diff
@@ -68,5 +68,19 @@
     "identity",
     "messageTeamMembers"
```
Comment (Prithvi-MSFT) [inline samples/bot-conversation/csharp/AppManifest/manifest.json:69]: Please change updated date for sample.json inside assets folder

**Review by Prithvi-MSFT — CHANGES_REQUESTED** [review]

File: samples/bot-conversation/csharp/Bots/TeamsConversationBot.cs [hunk 4]
```diff
@@ -72,6 +82,58 @@ protected override async Task OnTeamsMembersAddedAsync(IList<TeamsChannelAccount
...
+            if (teamMemberDetails.Count != 0)
+            {
+                var userList = string.Join(", ", users);
+                await turnContext.SendActivityAsync(MessageFactory.Text($"Number of members read the message : {_counter} \n\n Members : {userList}"), cancellationToken);
+            }
+            else
+            {
+                await turnContext.SendActivityAsync(MessageFactory.Text("Read count is zero. Please make sure to send message to all members firstly to check the count of members who have read your message."), cancellationToken);
+            }
+
```
Comment (ChetanSharma-msft) [inline samples/bot-conversation/csharp/Bots/TeamsConversationBot.cs:102]: Remove empty line

**Review by ChetanSharma-msft — APPROVED** [review]
Approved with one comment

File: samples/bot-conversation/csharp/Bots/TeamsConversationBot.cs (hunk 4 above)
Comment (Jegadeesh-MSFT) [reply samples/bot-conversation/csharp/Bots/TeamsConversationBot.cs]: Fixed

File: samples/bot-conversation/csharp/Bots/TeamsConversationBot.cs (hunk 2 above)
Comment (Jegadeesh-MSFT) [reply samples/bot-conversation/csharp/Bots/TeamsConversationBot.cs:248]: Fixed

File: samples/bot-conversation/csharp/AppManifest/manifest.json (hunk 3 above)
Comment (Jegadeesh-MSFT) [reply samples/bot-conversation/csharp/AppManifest/manifest.json:69]: Fixed

**Review by Pawank-MSFT — APPROVED** [review]
looks correct, Approving!

**Comment by Pawank-MSFT** [comment]
Executed Build Pipeline:
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/6321446697

image

**Review by Prithvi-MSFT — APPROVED** [review]
changes look fine

**Pull request merged**